Composer/packages/extensions/form-dialogs/src/app/@types/index.d.ts

```diff
@@ -1,16 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider doing this as a single line suppression to discourage future use of any in this file.

Reply:
we don't even need it anymore